lib/tbd/psi.rb

```diff
@@ -284,6 +292,7 @@ def initialize
         parapet:      0.500000, # "parapet" edge
         roof:         0.650000, # (non-parapet) "roof edge"
         fenestration: 0.505000, # "wall to vertical fenestration intersection"
+        spandrel:     0.000000, # (unspecified, defaults to 0)
```

Comment:
Quite a few edge types are unspecified in ASHRAE 90.1 2022 (A10) PSI sets, e.g. **:corners**. Suspect this is largely linked to the scarcity of published PSI factors for such intersections. There are usually 2 options in sorting the issue in energy codes:

- set 0 W/K.m for such edges in both _proposed design_ & _budget building design_
- set a non-0 W/K.m in both _proposed design_ & _budget building design_

In both cases, the effect is cancelled out; the _proposed design_ is neither advantaged or penalized (when compared to the _budget building design_). The second option remains a tad more realistic (e.g. for HVAC sizing).